### ISSUE TITLE
 Code and Paper don't seem to match...

### ISSUE THREAD

**Issue by wjddyd66** [idstcv/SoftTriple#10]
When mapped with Code, I am thinking as follows.

![image](https://user-images.githubusercontent.com/37298751/104862672-1009f380-5977-11eb-9d8c-ce493d32cacf.png)

In Paper... 
![image](https://user-images.githubusercontent.com/37298751/104863009-295f6f80-5978-11eb-9ce8-375917d778d8.png)

But, I think this Code is
![image](https://user-images.githubusercontent.com/37298751/104863027-39774f00-5978-11eb-83e7-da15d34f1199.png)

<code>lossClassify = F.cross_entropy(self.la*(simClass-marginM), target)</code>
->
**<code>lossClassify = F.cross_entropy(F.softmax(self.la * (simClass-marginM), dim=1), target)</code>**

**Comment by qian-qi** [comment]
Please check the official document of PyTorch about cross_entropy loss, which already includes the SoftMax operator.

**Comment by wjddyd66** [comment]
Sorry. I checked the Torch official website and confirmed that it includes the SoftMax operator!
